actor/src/lib.rs

```diff
@@ -163,7 +163,7 @@ impl<A: Actor> Addr<A> {
         tokio::task::spawn(async move {
             // TODO: propagate the error.
             if let Err(error) = sender.send(envelope).await {
-                iroha_logger::error!("Error sending actor message: {}", &error);
+                iroha_logger::error!(err = %error, "Error sending actor message");
```

Comment:
```suggestion
                iroha_logger::error!(%error, "Error sending actor message");
```
tracing uses a [shorthand notation](https://docs.rs/tracing/0.1.26/tracing/#recording-fields) for recording `value = value` kinds of fields.

Reply:
I know, I deliberately used full notation because change in local variable would change `field` name in log and this may not be intended. It's possible some log parsing code would depend on log API in the future

Reply:
It's a valid point. If there is any such use case, I think we should consider using the full notation when the old name and the new name have already diverged. It is a [premature optimisation](http://cultureandcommunication.org/galloway/premature-optimization-is-the-root-of-all-evil)

Reply:
would you still have me use shorthand notation?

Reply:
Up to you. This is an insignificant point, and there are still a few comments other than for using the shorthand notation.